### PR TITLE
Adding unit test for Active Benefit Cards (Header and Details)

### PR DESCRIPTION
**Pull request by amirpourpanahi**
### Description

List of proposed changes:

- Adding unit test for ActiveBenefitDetails component
- Adding unit test BenefitCardHeaderActive component

### CODE REVIEW

**Review by nabil386 — APPROVED** [review]

File: components/atoms/BenefitCardHeaderActive.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ export default function BenefitCardHeaderActive(props) {
   const t = props.locale === 'en' ? en : fr
 
   function viewPaymentHistory() {
-    console.log('View payment history button is Clicked')
+    window.open('./dashboard', 'ViewMyPayments')
```
Comment (arsanig) [inline components/atoms/BenefitCardHeaderActive.js:10]: How come this was changed from `console.log()`?

Comment (amirpourpanahi) [reply components/atoms/BenefitCardHeaderActive.js:10]: I removed <a></a> and used window.open() to link to dashboard page.
console.log() had been added to show button works. We don't need it anymore. Click on button refresh the page.

Comment (arsanig) [reply components/atoms/BenefitCardHeaderActive.js:10]: Can't you use the Next's router instead of `window.open()`?

Comment (amirpourpanahi) [reply components/atoms/BenefitCardHeaderActive.js]: I removed window,open and used Next's router

**Pull request merged**